initializr-service/src/main/resources/application.yml

```diff
@@ -22,6 +22,11 @@ spring:
 initializr:
   env:
     boms:
+      azure:
+        groupId: com.microsoft.azure
+        artifactId: azure-spring-boot-starter-bom
+        versionProperty: azure-bom.version
```

Comment:
I'd call that `azure.version`. I don't see any other Azure-related artifacts on the site.